x-pack/plugins/security/server/config.ts

```diff
@@ -120,6 +129,38 @@ const providersConfigSchema = schema.object(
         schema.object({ ...getCommonProviderSchemaProperties(), realm: schema.string() })
       )
     ),
+    anonymous: getUniqueProviderSchema(
+      'anonymous',
+      {
+        description: schema.string({
```

Comment:
__note:__ used description/hint/icon provided in the mockups as defaults, but we can discuss which of them we should keep as defaults.